hack/helm_vars/wire-server/values.yaml

```diff
@@ -235,8 +235,10 @@ spar:
 
 federator:
   replicaCount: 1
+  resources:
+    requests: {}
   imagePullPolicy: Always
   config:
     optSettings:
```

Comment:
for CI, shouldn't this set useSystemCAStore to false?

Reply:
Yes, I think you're right. Also, should the default assignment `useSystemCAStore: true` reside in `values.yaml` or in the template?